addons/binding/org.openhab.binding.ihc/src/main/java/org/openhab/binding/ihc/internal/ws/projectfile/IhcEnumValue.java

```diff
@@ -0,0 +1,41 @@
+/**
+ * Copyright (c) 2010-2019 Contributors to the openHAB project
+ *
+ * See the NOTICE file(s) distributed with this work for additional
+ * information.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ */
+package org.openhab.binding.ihc.internal.ws.projectfile;
+
+/**
+ * Class to store IHC / ELKO LS controller's enum value information.
+ *
+ * @author Pauli Anttila - Initial contribution
+ */
+public class IhcEnumValue {
+    private int id;
```

Comment:
I would propose to make these fields `final`

Reply:
Done